bin/cml/runner.js

```diff
@@ -245,63 +273,25 @@ const runLocal = async (opts) => {
   proc.stderr.on('data', dataHandler);
   proc.stdout.on('data', dataHandler);
   proc.on('disconnect', () =>
-    shutdown({ ...opts, reason: `runner disconnected` })
-  );
-  proc.on('close', (exit) =>
-    shutdown({ ...opts, reason: `runner closed with exit code ${exit}` })
+    shutdown({ ...opts, error: new Error('runner disconnected') })
```

Comment:
I believe this event only triggers if we call `subprocess.disconnect()` ourselves? and does not necessarily mean `"runner disconnected"` I feel that message has a network connotation and not an `IPC` channel has stopped sending messages.

Reply:
No, disconnected happens when by any meaning we do not have access to the runner's IPC channel

Reply:
How about this?
```suggestion
    shutdown({ ...opts, error: new Error('runner proccess lost') })
```

If I had an issue and looking at logs and saw an error `runner disconnected` my mind goes to networking not IPC

Reply:
Ah I see!